d2l-button-icon.js

```diff
@@ -4,15 +4,20 @@ Polymer-based web component for icon buttons
 
 @demo demo/button-icon.html d2l-button-icon
 */
-import '../@polymer/polymer/polymer-legacy.js';
+/*
+  FIXME(polymer-modulizer): the above comments were extracted
+  from HTML and may be out of place here. Review them and
+  then delete this comment!
+*/
```

Comment:
They're added by the modulizer and we haven't been removing them otherwise we'll end up with a patch file for every repo. I think it's best to leave them and then remove them up during the cleanup phase later.

Reply:
Oh I ended up removing them as part of my PR - good to know going forward